lib/tasks/webpacker.rake

```diff
@@ -97,6 +97,43 @@ namespace :webpacker do
 
       exec './bin/yarn add --dev typescript ts-loader && ./bin/yarn add "core-js zone.js rxjs @angular/core @angular/common @angular/compiler @angular/platform-browser @angular/platform-browser-dynamic"'
     end
+
+    desc "Install everything needed for Vue"
+    task :vue do
+      config_path = Rails.root.join('config/webpack/shared.js')
+      config = File.read(config_path)
+
+      # Module resolution https://webpack.js.org/concepts/module-resolution/
+      if config.include?("'vue$':'vue/dist/vue.common.js'")
```

Comment:
I think this won't be necessary anymore with Vue v2.2.x https://github.com/vuejs/vue/releases/tag/v2.2.0